docs/topics/data-analysis/data-science-libraries.md

```diff
@@ -1,88 +1,40 @@
 [//]: # (title: Kotlin and Java libraries for data analysis)
 
-Kotlin has 100% interoperability with Java, providing excellent performance and the ability to leverage 
-the entire ecosystem of tried and true Java libraries.
+From data collection to model building, Kotlin offers robust libraries facilitating 
+different tasks in the data pipeline.
 
-## Kotlin libraries
-
-* [Kotlin DataFrame](https://github.com/Kotlin/dataframe) is a library for structured data processing. It aims to
-  reconcile Kotlin's static typing with the dynamic nature of data by utilizing both the full power of the Kotlin language
-  and the opportunities provided by intermittent code execution in Jupyter notebooks and REPLs.
-
-* [Kandy](https://kotlin.github.io/kandy/welcome.html) is an open-source plotting library for the JVM written in Kotlin.
-  It provides a powerful and flexible DSL for chart creation,
-  along with seamless integration with [Kotlin Notebook](https://plugins.jetbrains.com/plugin/16340-kotlin-notebook)
-  and [Kotlin DataFrame](https://kotlin.github.io/dataframe/gettingstarted.html).
-
-* [Multik](https://github.com/Kotlin/multik) is a library for multidimensional arrays in Kotlin. The library provides Kotlin-idiomatic, 
-  type- and dimension-safe API for mathematical operations over multidimensional arrays. Multik offers swappable 
-  JVM and native computational engines, and a combination of the two for optimal performance.
-
-* [KotlinDL](https://github.com/jetbrains/kotlindl) is a high-level Deep Learning API written in Kotlin and inspired
-  by Keras. It offers simple APIs for training deep learning models from scratch, importing existing Keras models
-  for inference, and leveraging transfer learning for tweaking existing pre-trained models to your tasks.
-
-* [Kotlin for Apache Spark](https://github.com/JetBrains/kotlin-spark-api) adds a missing layer of compatibility between
-  Kotlin and Apache Spark. It allows Kotlin developers to use familiar language features such as data classes, and
-  lambda expressions as simple expressions in curly braces or method references.
+In addition, Kotlin has 100% interoperability with Java. This interoperability provides the ability to leverage 
```

Comment:
(Optional: just a thought to further highlight why this interoperability is so amazing - not only does the user get to use loads of Kotlin libraries but they can use Java as well)

In addition to its own libraries, Kotlin has 100% interoperability with Java.

Reply:
Added it. What do you think?

Reply:
lgtm 👍